packages/element-app/package.json

```diff
@@ -5,25 +5,28 @@
   "homepage": "https://element-did.com",
   "license": "Apache-2.0",
   "dependencies": {
-    "@material-ui/core": "^3.9.3",
+    "@material-ui/core": "^4.9.0",
     "@material-ui/icons": "^3.0.2",
-    "@transmute/did-wallet": "0.0.0-5",
+    "@transmute/did-wallet": "0.0.0-6",
     "@transmute/element-lib": "^0.4.3",
     "axios": "^0.18.0",
     "base64url": "^3.0.1",
     "classnames": "^2.2.6",
+    "clsx": "^1.0.4",
     "connected-react-router": "^6.2.1",
+    "crypto-ld": "^3.7.0",
+    "did-method-key": "^0.3.0",
```

Comment:
unused, we should be careful about adding more crypto dependencies to our already huge app bundle

Reply:
great catch, I had to add these as part of the keystore upgrade, forgot to remove them.